examples/Atmos/heldsuarez.jl

```diff
@@ -80,8 +80,8 @@ function run(mpicomm, polynomialorder, numelem_horz, numelem_vert,
   # determine the time step
   element_size = (setup.domain_height / numelem_vert)
   acoustic_speed = soundspeed_air(FT(315))
-  lucas_magic_factor = 14
-  dt = lucas_magic_factor * element_size / acoustic_speed / polynomialorder ^ 2
+  lucas_magic_factor = 5
+  dt = lucas_magic_factor * min_node_distance(grid) / acoustic_speed
```

Comment:
@mwarusz @lcw is there rationale on how this factor comes about?

Reply:
In practice, I set this factor as large as possible experimentally.  That is, I keep increasing it until the scheme goes unstable and then back off.

If we had an ODE solver that did error control then it would figure out this factor for us.  Otherwise, you might be able to do a linear stability analysis (such as in <doi:10.1016/j.jcp.2008.07.026>) to give you a better estimate.

Reply:
Once you have the `min_node_distance`, this factor is the Courant number (call it that). Make sure that when you use 1D IMEX, the `min_node_distance` here is just the *horizontal* minimum distance.